content/scout/scores.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Docker Scout health scores
+description: |
+  Docker Scout health scores provide a supply chain assessment for Docker Hub
+  images, grading them from A to F based on various security policies.
+keywords: scout, health scores, evaluation, checks, grades, docker hub
+sitemap: false
+---
+
+> **Early Access**
+>
+> Health scores is an [Early Access](/release-lifecycle/#early-access-ea)
+> feature of Docker Scout. The feature is only available to organizations
+> participating in the early access program for this feature.
+{ .restricted }
+
+Docker Scout health scores provide a security assessment, and overall supply
+chain health, of images on Docker Hub, helping you determine whether an image
+meets your security standards. The scores range from A to F, where A represents
+the highest level of security and F the lowest, offering an at-a-glance view of
+the security posture of your images.
+
+## Scoring system
+
+Health scores are determined by evaluating images against predefined,
+[out-of-the-box policies](./policy/_index.md#out-of-the-box-policies) provided
+by Docker Scout. Each policy is assigned a points value. If the image is
+compliant with a policy, it is awarded the points value for that policy. The
+health score of an image is calculated based on the percentage of points
+achieved relative to the total possible points.
+
+### Scoring process
+
+1. Each image is evaluated for compliance with the out-of-the-box policies.
+2. Points are awarded based on adherence to these policies.
+3. The points achieved percentage is calculated:
+
+   ```text
+   Percentage = (Points / Total) * 100
+   ```
+
+4. The final score is assigned based on the percentage of points achieved, as
+   shown in the following table:
+
+   | Points percentage (awarded out of total) | Score |
+   | ---------------------------------------- | ----- |
+   | More than 90%                            | A     |
+   | 71% to 90%                               | B     |
+   | 51% to 70%                               | C     |
+   | 31% to 50%                               | D     |
+   | 11% to 30%                               | E     |
+   | Less than 10%                            | F     |
+
+### Policy weights
+
+The out-of-the-box policies and their respective weights are as follows:
```

Comment:
I may be misremembering, but was there a conversation about not publishing all the details of the algorithm @cdupuis ?